roda-common/roda-common-data/src/main/java/org/roda/core/data/common/RodaConstants.java

```diff
@@ -730,6 +730,7 @@ public enum OrchestratorType {
 
   public static final String INDEX_INSTANCE_ID = "instanceId";
   public static final String INDEX_INSTANCE_NAME = "instanceName";
+  public static final String INDEX_CREATION_DATE = "creationDate";
```

Comment:
Please, to maintain consistency with the other collections, create a new constant for each collection with the text "createdOn"